migration/migrator/migrations/master/20240805165505_user_verification_code.py

```diff
@@ -0,0 +1,35 @@
+"""Migration for the Submitty master database."""
+
+
+def up(config, database):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    """
+
+    database.execute('''
+        CREATE TABLE if not exists public.unverified_users (
+            user_id character varying NOT NULL,
+            user_givenname character varying NOT NULL,
+            user_password character varying,
+            user_familyname character varying NOT NULL,
+            user_email character varying NOT NULL,
+            verification_code character varying(50) NOT NULL DEFAULT 'none',
```

Comment:
Instead of a magic value `none`, why not use `NULL` instead?  Is there a reason why this could ever be none/null in the first place?